### DISCUSSION TITLE
$wire.function return content to x-data

### DISCUSSION THREAD

**Discussion by dgironella** [livewire/livewire#4603] [1. Help]
Hi,

I try to use content returned by livewire function, to populate x-data content using code like this:

`x-data="{pdata : $wire.showPla}"`

I use with 

` <div x-text="pdata"></div>`

pdata is filled with content from livewire component function, but this is constantly making request to server, not one request and stop.

Thanks.

**Comment by joshhanley** [comment]
@dgironella I would probably do that in `x-init` rather than `x-data` to stop it running over and over.

Hope this helps!
